tests/testImageDecorrelation.py

```diff
@@ -84,17 +86,17 @@ def makeFakeImages(xim=None, yim=None, sig1=0.2, sig2=0.2, psf1=2.2, psf2=3.3, o
     """
     np.random.seed(seed)
 
-    psf1 = [2.2, 2.2] if psf1 is None else psf1
+    psf1 = [3.3, 3.3] if psf1 is None else psf1
```

Comment:
This won't do what the docstring says, since the default values are numbers instead of None.